src/ol_dbt/models/intermediate/ocw/int__ocw__resources.sql

```diff
@@ -10,10 +10,21 @@ with websites as (
     select * from {{ ref('stg__ocw__studio__postgres__websites_websitestarter') }}
 )
 
+, sitemetadata as (
+    select
+        website_uuid
+        , nullif(
+            json_query(websitecontent_metadata, 'lax $.primary_course_number' omit quotes), ''
+        ) as sitemetadata_primary_course_number
+        , nullif(json_query(websitecontent_metadata, 'lax $.term' omit quotes), '') as sitemetadata_course_term
+        , nullif(json_query(websitecontent_metadata, 'lax $.year' omit quotes), '') as sitemetadata_course_year
```

Comment:
Just FYI, these fields are already extracted in the staging, but up to you
```suggestion
         , nullif(
            course_primary_course_number, ''
        ) as sitemetadata_primary_course_number
        , course_term as sitemetadata_course_term
        , course_year as sitemetadata_course_year
```
```